homeassistant/components/samsungtv/media_player.py

```diff
@@ -196,12 +196,15 @@ async def async_update(self) -> None:
         """Update state of device."""
         if self._auth_failed or self.hass.is_stopping:
             return
+        old_state = self._attr_state
         if self._power_off_in_progress():
             self._attr_state = STATE_OFF
         else:
             self._attr_state = (
                 STATE_ON if await self._bridge.async_is_on() else STATE_OFF
             )
+        if self._attr_state != old_state:
+            LOGGER.info("TV %s state updated to %s", self._host, self._attr_state)
```

Comment:
As the purpose of this log line is for _debugging_ this line should be written to the debug channel i.s.o. info

Reply:
That was my original thought, but then I figured that this would follow more closely with the way coordinators work.
https://developers.home-assistant.io/docs/integration_quality_scale_index/#silver-
> Handles device/service unavailable. Log a warning once when unavailable, log once when reconnected.

I'm happy to change it if others agree with you.

Reply:
The line you've quoted from the quality scale index is about handling _unavailability_ though. Your proposed change is about logging every (On/Off) state change. Thats just a bit too verbose for info level logging I.M.O.